store.go

```diff
@@ -628,12 +624,6 @@ func (s *store) PutLayer(id, parent string, names []string, mountLabel string, w
 	if modified, err := rlstore.Modified(); modified || err != nil {
 		rlstore.Load()
 	}
-	ristore.Lock()
-	defer ristore.Unlock()
-	defer ristore.Touch()
```

Comment:
I think this removal is OK too, but don't know for sure.  I'm just not sure if the Touch should be removed here.